test/acceptance/features/environment.py

```diff
@@ -13,13 +13,24 @@
 """
 
 import subprocess
-from pyshould import should
+import os
+
+
+def before_all(_context):
+    start_sbo = os.getenv("TEST_ACCEPTANCE_START_SBO")
+    assert start_sbo is not None, "TEST_ACCEPTANCE_START_SBO is not set. It should be one of local, remote or operator-hub"
+    assert start_sbo in {"local", "remote", "operator-hub"}, "TEST_ACCEPTANCE_START_SBO should be one of local, remote or operator-hub"
+
+    if start_sbo == "local":
+        assert not os.getenv("TEST_ACCEPTANCE_SBO_STARTED").startswith("FAILED"), "TEST_ACCEPTANCE_SBO_STARTED shoud not be FAILED."
+    elif start_sbo == "remote":
+        sbo_namespace = os.getenv("SBO_NAMESPACE")
+        assert sbo_namespace is not None, "SBO_NAMESPACE is required but not set."
+        _context.sbo_namespace = sbo_namespace
+    elif start_sbo == "operator-hub":
+        return False
```

Comment:
I would fail here - with some assertion or something, letting developer that `operator-hub` case is still unsupported.

Reply:
Good point